internal/xds/translator/jsonpatch.go

```diff
@@ -51,6 +55,19 @@ func processJSONPatches(tCtx *types.ResourceVersionTable, envoyPatchPolicies []*
 				err          error
 			)
 
+			switch p.Operation.Op {
+			case AddOperation, ReplaceOperation:
+				if p.Operation.Value == nil {
+					status.SetEnvoyPatchPolicyInvalid(e.Status, "The add/replace operation required a value")
```

Comment:
lets add a Sprintf here to specify exact operation - `%s operation requires a value`